cmd/mdatagen/main_test.go

```diff
@@ -32,6 +32,7 @@ metrics:
     extended_documentation: Additional information on CPU Time can be found [here](https://en.wikipedia.org/wiki/CPU_time).
     unit: s
     sum:
+      number_type: double
```

Comment:
Is the number type a characteristic of a sum or gauge? Seems ok either way, but arguably it is an independent detail and could be specified at the top level.

Reply:
This field is applicable to Sum and Gauge types only, not to Histogram. That's why I think if makes sense to put it under the the type section.

Reply:
Makes sense. 👍